scripts/publish-npm-packages.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash
+
+# Set default values for dist-tag and registry for local development
+# to prevent running this script and accidentally publishing to npm
+dist_tag="canary"
+registry="http://localhost:4873"
+
+# shellcheck source=./scripts/helpers/exit-if-fail.sh
+source "$(dirname "$0")/helpers/exit-if-fail.sh"
+
+if [ -z "$NPM_TOKEN" ]; then
```

Comment:
@jackw would it make sense no to require an npm token when the registry is verdaccio?

Reply:
I'm not too sure as this is used for publishing the canary packages to NPM registry in CI and if someone were to remove the token in CI this would make it easy for anyone to spot the issue rather than getting a load of auth errors.

I also believe that verdaccio / npm requires a token regardless. I think I had to add `//localhost:4873/:_authToken="foobar"` to my npmrc for local publishing to work with verdaccio. 🤔